src/ui/sass/modules/_Pagination.scss

```diff
@@ -55,4 +55,10 @@
   {
     margin-right: -4px;
   }
+
+  &-chevron,
+  &-doubleChevron
+  {
+    display: flex;
```

Comment:
Is vertical centering an issue for the left chevrons as well?

Reply:
The left chevron just applies a reversal to the right chevron, iirc.

Reply:
Oof good catch, I did not realize that the left chevrons had different class names. Updated to add them here!